src/commands/synthetics/interfaces.ts

```diff
@@ -328,6 +328,7 @@ export interface Suite {
 
 export interface Summary {
   batchId?: string
+  batchUrl?: string
```

Comment:
Passing a base URL is definitely not ideal but I feel like we're overloading the summary a bit beyond its purpose. Since we also have the same issue with `resultEnd` how about exposing more straightforward utility functions like `getBatchUrl(config, batchId)` and `getResultUrl(config, result)` that can be use by library users